receiver/statsdreceiver/config.go

```diff
@@ -73,6 +73,19 @@ func (c *Config) Validate() error {
 				errs = multierr.Append(errs, fmt.Errorf("histogram configuration requires observer_type: histogram"))
 			}
 		}
+		if eachMap.ObserverType == protocol.SummaryObserver {
+			if len(eachMap.Summary.Percentiles) != 0 {
```

Comment:
Can we move this check before `if eachMap.ObserverType == protocol.SummaryObserver` to not repeat it?